packages/kbn-sort-predicates/src/sorting.ts

```diff
@@ -10,6 +10,7 @@ import versionCompare from 'compare-versions';
 import valid from 'semver/functions/valid';
 import ipaddr, { type IPv4, type IPv6 } from 'ipaddr.js';
 import { FieldFormat } from '@kbn/field-formats-plugin/common';
+import moment from 'moment';
```

Comment:
I've considered using something like `date-fns` to reduce the footprint of this package, but due to the parsing difference between `moment` and `date-fns` I've preferred consistency over bundle size in this case.